site/_data/docs/privacy-sandbox/toc.yml

```diff
@@ -1,11 +1,39 @@
-- url: /docs/privacy-sandbox/overview
-- url: /docs/privacy-sandbox/cds21-update
-- url: /docs/privacy-sandbox/status
-- url: /docs/privacy-sandbox/faq
-- url: /docs/privacy-sandbox/glossary
-- url: /docs/privacy-sandbox/trust-tokens
-- url: /docs/privacy-sandbox/first-party-sets
-- url: /docs/privacy-sandbox/attribution-reporting
-- url: /docs/privacy-sandbox/user-agent
-- url: /docs/privacy-sandbox/floc
-- url: /docs/privacy-sandbox/fledge
+# External URLs need an explicit title and description, otherwise
+# the toc won't know what to display.
+- title: i18n.docs.privacy-sandbox.overview
+  sections:
+    - url: /docs/privacy-sandbox/overview
+      title: i18n.docs.privacy-sandbox.overview-article
+    - url: /docs/privacy-sandbox/cds21-update
```

Comment:
CDS update should be the last item in the intro section, as we should aim to break this up and create the more general process guidance, then the CDS update remains for historical reference but won't be a top-level item.

Reply:
I imagine it transforming into "Collaboration" and dropping the CDS-specific context. But that said, I can move it for now as it remains CDS-specific.